### ISSUE TITLE
Consider adding oauth token handling

### ISSUE THREAD

**Issue by alkaris2** [pystardust/ani-cli#191]
It appears when trying to download an episode from GoGoAnime you can no longer do so, because GoGoAnime now requires logins to download episodes, so even if you do attempt to download an episode for something, it'll only return an error saying that it's not available. What could do instead is implement a user login token, perhaps with OAuth or similar, the user would then have to have/create an account to download the episodes they want.

**Comment by dconixDev** [comment]
Not familiar with the code base but doesnt this line
```ffmpeg -headers "Referer: $embedded_video_url" -i "$video_url" \
				-c copy "${anime_id}-${episode}.mkv" >/dev/null 2>&1 &&
				printf "${c_green}Downloaded episode: %s${c_reset}\n" "$episode" ||
				printf "${c_red}Download failed episode: %s${c_reset}\n" "$episode"```
copy the stream to a file or am I misunderstanding how ffmpeg works?

**Comment by ahmadjehad** [comment]
even if it says "downloaded" the episode is about 1kb and don't open, tell me if it works for you guys

**Comment by ura43** [comment]
I believe your issue was either solved with #209 or #232 
If thats not the case, please reopen the issue
